assets/helpers/checkouts.js

```diff
@@ -72,7 +73,11 @@ function getValidContributionTypesFromUrlOrElse(fallback: ContributionType[]): C
   return fallback;
 }
 
-function getValidContributionTypes(): ContributionType[] {
+function getValidContributionTypes(abParticipations: Participations): ContributionType[] {
+  const { globalContributionTypes } = abParticipations;
+  if (globalContributionTypes === 'default-annual_no-monthly') {
+    return ['ONE_OFF', 'ANNUAL'];
+  }
   return getValidContributionTypesFromUrlOrElse(['ONE_OFF', 'MONTHLY', 'ANNUAL']);
```

Comment:
personally I'd make the test one the `OrElse` - not sure if anyone's using the URL mechanism yet but I'd expect it to override tests